### DISCUSSION TITLE
Key overlay should be in view even when hud is hidden

### DISCUSSION THREAD

**Discussion by HachikoG45** [ppy/osu#23323] [Ideas] [answered]
![Screenshot_osu!_2023-04-26-21-53-30-952](https://user-images.githubusercontent.com/93947726/234763004-9777f9bc-98ff-476e-ad18-d38c7c9ab9db.jpg) 
![Screenshot_osu!_2023-04-26-21-54-08-713](https://user-images.githubusercontent.com/93947726/234763793-ca1ccad6-c389-43d8-8f18-9d05907f1ecb.jpg)

 I think key overlay should always be in view even if hud is hidden, key overlay even if it's part of hud should be taken as separate

**Comment by Joehuu** [comment, marked as answer]
Already tracked as an issue in https://github.com/ppy/osu/issues/12039.
